client/src/main/scala/scripts/ExercisesJS.scala

```diff
@@ -1,7 +1,5 @@
 /*
- *  scala-exercises
- *
- *  Copyright 2015-2017 47 Degrees, LLC. <http://www.47deg.com>
+ * Copyright 2016-2019 47 Degrees, LLC. <http://www.47deg.com>
```

Comment:
It is 2015 everywhere, why change it to 2016 here?

Reply:
In build.sbt in the copyright part says 2015 but the `startDate` option says 2016. Which one is correct?